v1/examples/pipeline/stages/cd/service_env_variations/env-with-deploy-all-filter.yaml

```diff
@@ -0,0 +1,28 @@
+version: 1
+kind: pipeline
+spec:
+  stages:
+    - type: deployment
+      failure:
+        - errors:
+            - all
+          action:
+            type: fail
+      spec:
+        services:
+          inherit:
+            stage: pre-qa
```

Comment:
pre-qa is identifier of a stage in the same pipeline, right?
can we either add a stage with this id in this pipeline or a comment explaining the same to make it clear?